openstack/ironic/values.yaml

```diff
@@ -31,6 +31,36 @@ debug: "True"
 
 kos_conductor: true
 
+cert_robot:
+  image_tag: "latest"
+  schedule: "@weekly"
+  env:
+    issuer:
+    dns_resolvers:
+    acme_server:
+    csr:
+      country:
+      state:
+      city:
+      organization:
+      organizational_unit:
+    os_cloud: default  # Needs to match the name of the cloud in cloud_common and cloud_secure
+  cloud_common:
+    clouds:
+      default:
+        auth:
+          auth_url: http://keystone:5000/v3
```

Comment:
Does this get overwritten with a sensible value by default?

Reply:
It doesn't get overwritten, and should work out of the box by relying on some conventions:
Ironic and Keystone get deployed in the same namespace, and the service for keystone has been named "keystone", and listens on port 5000.

It works together with our default dns resolution and the fact that keystone has been deployed in the same namespace as ironic.
Since it has no dots in the name, it will trigger a dns-search, and the first item in the list is `monsoon3.svc.kubernetes.<region>.cloud.sap`, so it expands to `keystone.monsoon3.svc.kubernetes.qa-de-2.cloud.sap`, which is declared [here](https://github.com/sapcc/helm-charts/blob/master/openstack/keystone/templates/service-api.yaml).